fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResourceSwagger.java

```diff
@@ -1126,23 +1126,23 @@ private PostLoansRequest() {}
         @Schema(example = "12")
         public Integer loanTermFrequency;
         @Schema(example = "2")
-        public Integer loanTermFrequencyType;
+        public Long loanTermFrequencyType;
         @Schema(example = "12")
         public Integer numberOfRepayments;
         @Schema(example = "1")
         public Integer repaymentEvery;
         @Schema(example = "2")
-        public Integer repaymentFrequencyType;
+        public Long repaymentFrequencyType;
         @Schema(example = "2")
         public Integer interestRatePerPeriod;
         @Schema(example = "1")
-        public Integer amortizationType;
+        public Long amortizationType;
         @Schema(example = "5.5")
         public BigDecimal fixedPrincipalPercentagePerInstallment;
         @Schema(example = "0")
-        public Integer interestType;
+        public Long interestType;
         @Schema(example = "1")
-        public Integer interestCalculationPeriodType;
+        public Long interestCalculationPeriodType;
```

Comment:
This looks wrong to me based on LoanScheduleAssembler line 215. It is the ordinal of an enum.

Reply:
Done.